### DISCUSSION TITLE
<title>I need advice on resolving issues with my GitHub account being flagged or suspended

### DISCUSSION THREAD

**Discussion by EzioNinja** [community/community#130959] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Everyone,

I'm reaching out to the community for advice on recovering my GitHub account, which was suspended two days ago. On Sunday night, I received an email stating that my GitHub account had been suspended. When attempting to reset my password, I encountered the message "Account suspended. Access to your account has been suspended due to a violation of our Terms of Service."

I am uncertain why this has occurred. GitHub is crucial for both my work and college commitments, including my current internship where all my tasks are managed through my GitHub account. Unfortunately, my mentor also cannot access the account.

My repositories containing work and college projects are stored there.

Given the importance of my GitHub account to my professional and academic life, I am urgently requesting reinstatement of my account.

I have been facing this issue for the past three days without receiving a response.

My ticket number is #**2865395**.
My GitHub Account Username :- **HarshalScripter**

Thank you for your assistance.

**Comment by asdacosta** [comment]
Same issue was resolved here:
https://github.com/orgs/community/discussions/58191


You can go through this to grasp:
https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement

**Comment by Mee-meet** [comment]
Subject: Request for Unsuspension of GitHub Account

Dear GitHub Support Team,

I hope this message finds you well. My name is Meet Mehta, and I am writing to request the unsuspension of my GitHub account. The details of my account are as follows:

Username: MeetInCode
Email: meets.studentlife@gmail.com
I recently discovered that my account has been suspended, and I am unsure of the reason for this action. I rely heavily on GitHub for my studies and projects, and losing access to my repositories has caused significant disruption.

If the suspension was due to a misunderstanding or an error on my part, I sincerely apologize and am willing to take any necessary corrective actions to ensure compliance with GitHub's terms of service.

I would greatly appreciate your assistance in reviewing my case and providing guidance on how to resolve this issue. Thank you for your understanding and support.

Looking forward to your prompt response.

Best regards,
Meet Mehta
